diff --git a/CMakeLists.txt b/CMakeLists.txt
index ae06ff493..c40d89f25 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -169,12 +169,9 @@
 # NOTE: update these lines by running `txstats.py`, or for full automation
 #       `txstats.py -e`. See also
 #
-# Total 82 languages
-set( _tx_complete ar az ca cs_CZ de en es es_AR fi_FI fr he hu id
-    ja kk ko lg nb pl pt_BR ru sv tg tr_TR uk zh_CN zh_TW )  # 27 languages
-set( _tx_good az_AZ bg fur hr is it_IT ka lt pt_PT sk sq uz )  # 12 languages
-set( _tx_ok as ast be bn ca@valencia da el en_GB eo es_MX et eu fa
-    gl hi ia ml mr nl oc ro si sl sr sr@latin th vi )  # 27 languages
+set( _tx_complete de en hu )
+set( _tx_good )
+set( _tx_ok en_GB )
 set( _tx_incomplete bs es_PR fr_CA gu ie ja-Hira kn lo lv mk ne_NP
     ta_IN te ur zh zh_HK )  # 16 languages
 # Total 82 languages
-- 
2.35.1
